ext_localconf.php

```diff
@@ -19,3 +19,172 @@
 $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS'][\TYPO3\CMS\Recordlist\RecordList\DatabaseRecordList::class]['modifyQuery'][] =
     \T3G\Querybuilder\Hooks\DatabaseRecordList::class;
 
+// Create DataProviderGroup
```

Comment:
Isn't this unused and can be deleted? I think I remember we removed this dependency ...

Reply:
confirmed and removed